scss/_patterns_status-label.scss

```diff
@@ -0,0 +1,52 @@
+@import 'settings';
+
+// Default status label styling
+@mixin vf-p-status-label {
```

Comment:
Nice, I haven't thought we should rename the file and mixin as well.

To avoid duplication between this file and old `p-label` one I think we should move the `p-label` definition here are reuse as much of the code as possible.

So `.p-label` can extend `%vf-status-label` as well, etc.

Reply:
Makes sense, thanks!